app/src/main/java/org/simlar/helper/ContactDataComplete.java

```diff
@@ -45,6 +45,6 @@ public String getNameOrNumber()
 	public char getFirstChar()
 	{
 		final String nameOrNumber = getNameOrNumber();
-		return Util.isNullOrEmpty(nameOrNumber) ? ' ' : nameOrNumber.charAt(0);
+		return Util.isNullOrEmpty(nameOrNumber) ? ' ' : Character.toUpperCase(nameOrNumber.charAt(0));
```

Comment:
Character.toTitleCase() because of [What is titlecase? How is it different from uppercase?](http://unicode.org/faq/casemap_charprop.html#4)